src/pyinfraboxutils/coverageTest.py

```diff
@@ -0,0 +1,49 @@
+#! /usr/bin/env python
```

Comment:
please move the test files to `infrabox/test/pyinfraboxutils` and create a Dockerfile which executes the tests. you can probably copy and adapt `infrabox/test/pyinfrabox/Dockerfile`

Reply:
Done in next commit, but not sure the dockerfile is correct